[2015-01-16 04:01:17] log.ERROR: exception 'Symfony\Component\HttpFoundation\File\Exception\FileException' with message 'Unable to create the "/var/www/trial/dropzone/public/uploads/" directory' in /var/www/trial/dropzone/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/File/File.php:134 Stack trace: #0 /var/www/trial/dropzone/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/File/UploadedFile.php(239): Symfony\Component\HttpFoundation\File\File->getTargetFile('/var/www/trial/...', 'images (1).jpg') #1 /var/www/trial/dropzone/app/routes.php(29): Symfony\Component\HttpFoundation\File\UploadedFile->move('/var/www/trial/...', 'images (1).jpg') #2 [internal function]: {closure}() #3 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #4 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #5 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #6 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #7 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #8 /var/www/trial/dropzone/public/index.php(49): Illuminate\Foundation\Application->run() #9 {main} [] []
[2015-01-16 04:01:17] log.ERROR: exception 'Symfony\Component\HttpFoundation\File\Exception\FileException' with message 'Unable to create the "/var/www/trial/dropzone/public/uploads/" directory' in /var/www/trial/dropzone/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/File/File.php:134 Stack trace: #0 /var/www/trial/dropzone/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/File/UploadedFile.php(239): Symfony\Component\HttpFoundation\File\File->getTargetFile('/var/www/trial/...', 'images (2).jpg') #1 /var/www/trial/dropzone/app/routes.php(29): Symfony\Component\HttpFoundation\File\UploadedFile->move('/var/www/trial/...', 'images (2).jpg') #2 [internal function]: {closure}() #3 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #4 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #5 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #6 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #7 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #8 /var/www/trial/dropzone/public/index.php(49): Illuminate\Foundation\Application->run() #9 {main} [] []
[2015-01-16 04:01:17] log.ERROR: exception 'Symfony\Component\HttpFoundation\File\Exception\FileException' with message 'Unable to create the "/var/www/trial/dropzone/public/uploads/" directory' in /var/www/trial/dropzone/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/File/File.php:134 Stack trace: #0 /var/www/trial/dropzone/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/File/UploadedFile.php(239): Symfony\Component\HttpFoundation\File\File->getTargetFile('/var/www/trial/...', 'images (4).jpg') #1 /var/www/trial/dropzone/app/routes.php(29): Symfony\Component\HttpFoundation\File\UploadedFile->move('/var/www/trial/...', 'images (4).jpg') #2 [internal function]: {closure}() #3 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #4 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #5 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #6 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #7 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #8 /var/www/trial/dropzone/public/index.php(49): Illuminate\Foundation\Application->run() #9 {main} [] []
[2015-01-16 04:01:19] log.ERROR: exception 'Symfony\Component\HttpFoundation\File\Exception\FileException' with message 'Unable to create the "/var/www/trial/dropzone/public/uploads/" directory' in /var/www/trial/dropzone/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/File/File.php:134 Stack trace: #0 /var/www/trial/dropzone/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/File/UploadedFile.php(239): Symfony\Component\HttpFoundation\File\File->getTargetFile('/var/www/trial/...', 'images (3).jpg') #1 /var/www/trial/dropzone/app/routes.php(29): Symfony\Component\HttpFoundation\File\UploadedFile->move('/var/www/trial/...', 'images (3).jpg') #2 [internal function]: {closure}() #3 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #4 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #5 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #6 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #7 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #8 /var/www/trial/dropzone/public/index.php(49): Illuminate\Foundation\Application->run() #9 {main} [] []
[2015-01-16 04:01:19] log.ERROR: exception 'Symfony\Component\HttpFoundation\File\Exception\FileException' with message 'Unable to create the "/var/www/trial/dropzone/public/uploads/" directory' in /var/www/trial/dropzone/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/File/File.php:134 Stack trace: #0 /var/www/trial/dropzone/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/File/UploadedFile.php(239): Symfony\Component\HttpFoundation\File\File->getTargetFile('/var/www/trial/...', 'images (5).jpg') #1 /var/www/trial/dropzone/app/routes.php(29): Symfony\Component\HttpFoundation\File\UploadedFile->move('/var/www/trial/...', 'images (5).jpg') #2 [internal function]: {closure}() #3 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #4 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #5 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #6 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #7 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #8 /var/www/trial/dropzone/public/index.php(49): Illuminate\Foundation\Application->run() #9 {main} [] []
[2015-01-16 04:01:19] log.ERROR: exception 'Symfony\Component\HttpFoundation\File\Exception\FileException' with message 'Unable to create the "/var/www/trial/dropzone/public/uploads/" directory' in /var/www/trial/dropzone/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/File/File.php:134 Stack trace: #0 /var/www/trial/dropzone/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/File/UploadedFile.php(239): Symfony\Component\HttpFoundation\File\File->getTargetFile('/var/www/trial/...', 'images (7).jpg') #1 /var/www/trial/dropzone/app/routes.php(29): Symfony\Component\HttpFoundation\File\UploadedFile->move('/var/www/trial/...', 'images (7).jpg') #2 [internal function]: {closure}() #3 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #4 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #5 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #6 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #7 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #8 /var/www/trial/dropzone/public/index.php(49): Illuminate\Foundation\Application->run() #9 {main} [] []
[2015-01-16 04:01:22] log.ERROR: exception 'Symfony\Component\HttpFoundation\File\Exception\FileException' with message 'Unable to create the "/var/www/trial/dropzone/public/uploads/" directory' in /var/www/trial/dropzone/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/File/File.php:134 Stack trace: #0 /var/www/trial/dropzone/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/File/UploadedFile.php(239): Symfony\Component\HttpFoundation\File\File->getTargetFile('/var/www/trial/...', 'images (6).jpg') #1 /var/www/trial/dropzone/app/routes.php(29): Symfony\Component\HttpFoundation\File\UploadedFile->move('/var/www/trial/...', 'images (6).jpg') #2 [internal function]: {closure}() #3 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #4 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #5 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #6 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #7 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #8 /var/www/trial/dropzone/public/index.php(49): Illuminate\Foundation\Application->run() #9 {main} [] []
[2015-01-16 04:01:42] log.ERROR: exception 'Symfony\Component\HttpFoundation\File\Exception\FileException' with message 'Unable to create the "/var/www/trial/dropzone/public/uploads/" directory' in /var/www/trial/dropzone/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/File/File.php:134 Stack trace: #0 /var/www/trial/dropzone/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/File/UploadedFile.php(239): Symfony\Component\HttpFoundation\File\File->getTargetFile('/var/www/trial/...', 'Habibi') #1 /var/www/trial/dropzone/app/routes.php(29): Symfony\Component\HttpFoundation\File\UploadedFile->move('/var/www/trial/...', 'Habibi') #2 [internal function]: {closure}() #3 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #4 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #5 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #6 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #7 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #8 /var/www/trial/dropzone/public/index.php(49): Illuminate\Foundation\Application->run() #9 {main} [] []
[2015-01-16 04:01:44] log.ERROR: exception 'Symfony\Component\HttpFoundation\File\Exception\FileException' with message 'Unable to create the "/var/www/trial/dropzone/public/uploads/" directory' in /var/www/trial/dropzone/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/File/File.php:134 Stack trace: #0 /var/www/trial/dropzone/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/File/UploadedFile.php(239): Symfony\Component\HttpFoundation\File\File->getTargetFile('/var/www/trial/...', 'kombinasi_perta...') #1 /var/www/trial/dropzone/app/routes.php(29): Symfony\Component\HttpFoundation\File\UploadedFile->move('/var/www/trial/...', 'kombinasi_perta...') #2 [internal function]: {closure}() #3 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #4 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #5 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #6 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #7 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #8 /var/www/trial/dropzone/public/index.php(49): Illuminate\Foundation\Application->run() #9 {main} [] []
[2015-01-16 04:04:30] log.ERROR: exception 'Symfony\Component\HttpFoundation\File\Exception\FileException' with message 'Unable to create the "/var/www/trial/dropzone/public/uploads/" directory' in /var/www/trial/dropzone/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/File/File.php:134 Stack trace: #0 /var/www/trial/dropzone/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/File/UploadedFile.php(239): Symfony\Component\HttpFoundation\File\File->getTargetFile('/var/www/trial/...', 'kombinasi_perta...') #1 /var/www/trial/dropzone/app/routes.php(29): Symfony\Component\HttpFoundation\File\UploadedFile->move('/var/www/trial/...', 'kombinasi_perta...') #2 [internal function]: {closure}() #3 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #4 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #5 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #6 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #7 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #8 /var/www/trial/dropzone/public/index.php(49): Illuminate\Foundation\Application->run() #9 {main} [] []
[2015-01-16 04:04:30] log.ERROR: exception 'Symfony\Component\HttpFoundation\File\Exception\FileException' with message 'Unable to create the "/var/www/trial/dropzone/public/uploads/" directory' in /var/www/trial/dropzone/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/File/File.php:134 Stack trace: #0 /var/www/trial/dropzone/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/File/UploadedFile.php(239): Symfony\Component\HttpFoundation\File\File->getTargetFile('/var/www/trial/...', 'Habibi..odp') #1 /var/www/trial/dropzone/app/routes.php(29): Symfony\Component\HttpFoundation\File\UploadedFile->move('/var/www/trial/...', 'Habibi..odp') #2 [internal function]: {closure}() #3 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #4 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #5 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #6 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #7 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #8 /var/www/trial/dropzone/public/index.php(49): Illuminate\Foundation\Application->run() #9 {main} [] []
[2015-01-16 04:04:38] log.ERROR: exception 'Symfony\Component\HttpFoundation\File\Exception\FileException' with message 'Unable to create the "/var/www/trial/dropzone/public/uploads/" directory' in /var/www/trial/dropzone/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/File/File.php:134 Stack trace: #0 /var/www/trial/dropzone/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/File/UploadedFile.php(239): Symfony\Component\HttpFoundation\File\File->getTargetFile('/var/www/trial/...', 'images (1).jpg') #1 /var/www/trial/dropzone/app/routes.php(29): Symfony\Component\HttpFoundation\File\UploadedFile->move('/var/www/trial/...', 'images (1).jpg') #2 [internal function]: {closure}() #3 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #4 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #5 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #6 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #7 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #8 /var/www/trial/dropzone/public/index.php(49): Illuminate\Foundation\Application->run() #9 {main} [] []
[2015-01-16 04:04:38] log.ERROR: exception 'Symfony\Component\HttpFoundation\File\Exception\FileException' with message 'Unable to create the "/var/www/trial/dropzone/public/uploads/" directory' in /var/www/trial/dropzone/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/File/File.php:134 Stack trace: #0 /var/www/trial/dropzone/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/File/UploadedFile.php(239): Symfony\Component\HttpFoundation\File\File->getTargetFile('/var/www/trial/...', 'images (2).jpg') #1 /var/www/trial/dropzone/app/routes.php(29): Symfony\Component\HttpFoundation\File\UploadedFile->move('/var/www/trial/...', 'images (2).jpg') #2 [internal function]: {closure}() #3 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #4 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #5 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #6 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #7 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #8 /var/www/trial/dropzone/public/index.php(49): Illuminate\Foundation\Application->run() #9 {main} [] []
[2015-01-16 04:04:38] log.ERROR: exception 'Symfony\Component\HttpFoundation\File\Exception\FileException' with message 'Unable to create the "/var/www/trial/dropzone/public/uploads/" directory' in /var/www/trial/dropzone/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/File/File.php:134 Stack trace: #0 /var/www/trial/dropzone/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/File/UploadedFile.php(239): Symfony\Component\HttpFoundation\File\File->getTargetFile('/var/www/trial/...', 'images (3).jpg') #1 /var/www/trial/dropzone/app/routes.php(29): Symfony\Component\HttpFoundation\File\UploadedFile->move('/var/www/trial/...', 'images (3).jpg') #2 [internal function]: {closure}() #3 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #4 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #5 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #6 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #7 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #8 /var/www/trial/dropzone/public/index.php(49): Illuminate\Foundation\Application->run() #9 {main} [] []
[2015-01-16 04:04:38] log.ERROR: exception 'Symfony\Component\HttpFoundation\File\Exception\FileException' with message 'Unable to create the "/var/www/trial/dropzone/public/uploads/" directory' in /var/www/trial/dropzone/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/File/File.php:134 Stack trace: #0 /var/www/trial/dropzone/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/File/UploadedFile.php(239): Symfony\Component\HttpFoundation\File\File->getTargetFile('/var/www/trial/...', 'images (4).jpg') #1 /var/www/trial/dropzone/app/routes.php(29): Symfony\Component\HttpFoundation\File\UploadedFile->move('/var/www/trial/...', 'images (4).jpg') #2 [internal function]: {closure}() #3 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #4 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #5 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #6 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #7 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #8 /var/www/trial/dropzone/public/index.php(49): Illuminate\Foundation\Application->run() #9 {main} [] []
[2015-01-16 04:04:38] log.ERROR: exception 'Symfony\Component\HttpFoundation\File\Exception\FileException' with message 'Unable to create the "/var/www/trial/dropzone/public/uploads/" directory' in /var/www/trial/dropzone/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/File/File.php:134 Stack trace: #0 /var/www/trial/dropzone/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/File/UploadedFile.php(239): Symfony\Component\HttpFoundation\File\File->getTargetFile('/var/www/trial/...', 'images (5).jpg') #1 /var/www/trial/dropzone/app/routes.php(29): Symfony\Component\HttpFoundation\File\UploadedFile->move('/var/www/trial/...', 'images (5).jpg') #2 [internal function]: {closure}() #3 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #4 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #5 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #6 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #7 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #8 /var/www/trial/dropzone/public/index.php(49): Illuminate\Foundation\Application->run() #9 {main} [] []
[2015-01-16 04:04:38] log.ERROR: exception 'Symfony\Component\HttpFoundation\File\Exception\FileException' with message 'Unable to create the "/var/www/trial/dropzone/public/uploads/" directory' in /var/www/trial/dropzone/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/File/File.php:134 Stack trace: #0 /var/www/trial/dropzone/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/File/UploadedFile.php(239): Symfony\Component\HttpFoundation\File\File->getTargetFile('/var/www/trial/...', 'images (6).jpg') #1 /var/www/trial/dropzone/app/routes.php(29): Symfony\Component\HttpFoundation\File\UploadedFile->move('/var/www/trial/...', 'images (6).jpg') #2 [internal function]: {closure}() #3 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #4 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #5 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #6 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #7 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #8 /var/www/trial/dropzone/public/index.php(49): Illuminate\Foundation\Application->run() #9 {main} [] []
[2015-01-16 04:04:38] log.ERROR: exception 'Symfony\Component\HttpFoundation\File\Exception\FileException' with message 'Unable to create the "/var/www/trial/dropzone/public/uploads/" directory' in /var/www/trial/dropzone/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/File/File.php:134 Stack trace: #0 /var/www/trial/dropzone/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/File/UploadedFile.php(239): Symfony\Component\HttpFoundation\File\File->getTargetFile('/var/www/trial/...', 'images (7).jpg') #1 /var/www/trial/dropzone/app/routes.php(29): Symfony\Component\HttpFoundation\File\UploadedFile->move('/var/www/trial/...', 'images (7).jpg') #2 [internal function]: {closure}() #3 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #4 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #5 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #6 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #7 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #8 /var/www/trial/dropzone/public/index.php(49): Illuminate\Foundation\Application->run() #9 {main} [] []
[2015-01-16 04:14:35] log.ERROR: exception 'Symfony\Component\HttpFoundation\File\Exception\FileException' with message 'Unable to create the "/var/www/trial/dropzone/public/uploads/" directory' in /var/www/trial/dropzone/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/File/File.php:134 Stack trace: #0 /var/www/trial/dropzone/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/File/UploadedFile.php(239): Symfony\Component\HttpFoundation\File\File->getTargetFile('/var/www/trial/...', 'images (2).jpg') #1 /var/www/trial/dropzone/app/routes.php(29): Symfony\Component\HttpFoundation\File\UploadedFile->move('/var/www/trial/...', 'images (2).jpg') #2 [internal function]: {closure}() #3 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #4 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #5 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #6 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #7 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #8 /var/www/trial/dropzone/public/index.php(49): Illuminate\Foundation\Application->run() #9 {main} [] []
[2015-01-16 04:14:35] log.ERROR: exception 'Symfony\Component\HttpFoundation\File\Exception\FileException' with message 'Unable to create the "/var/www/trial/dropzone/public/uploads/" directory' in /var/www/trial/dropzone/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/File/File.php:134 Stack trace: #0 /var/www/trial/dropzone/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/File/UploadedFile.php(239): Symfony\Component\HttpFoundation\File\File->getTargetFile('/var/www/trial/...', 'images (3).jpg') #1 /var/www/trial/dropzone/app/routes.php(29): Symfony\Component\HttpFoundation\File\UploadedFile->move('/var/www/trial/...', 'images (3).jpg') #2 [internal function]: {closure}() #3 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #4 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #5 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #6 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #7 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #8 /var/www/trial/dropzone/public/index.php(49): Illuminate\Foundation\Application->run() #9 {main} [] []
[2015-01-16 04:14:35] log.ERROR: exception 'Symfony\Component\HttpFoundation\File\Exception\FileException' with message 'Unable to create the "/var/www/trial/dropzone/public/uploads/" directory' in /var/www/trial/dropzone/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/File/File.php:134 Stack trace: #0 /var/www/trial/dropzone/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/File/UploadedFile.php(239): Symfony\Component\HttpFoundation\File\File->getTargetFile('/var/www/trial/...', 'images (4).jpg') #1 /var/www/trial/dropzone/app/routes.php(29): Symfony\Component\HttpFoundation\File\UploadedFile->move('/var/www/trial/...', 'images (4).jpg') #2 [internal function]: {closure}() #3 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #4 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #5 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #6 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #7 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #8 /var/www/trial/dropzone/public/index.php(49): Illuminate\Foundation\Application->run() #9 {main} [] []
[2015-01-16 06:20:35] log.ERROR: exception 'Intervention\Image\Exception\NotReadableException' with message 'Unable to read image from file (/var/www/trial/dropzone/public/uploads/evaluation-result-blok_1_-_learning_how_people_learn_1_.xlsx).' in /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/Gd/Decoder.php:21 Stack trace: #0 /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/AbstractDecoder.php(248): Intervention\Image\Gd\Decoder->initFromPath('/var/www/trial/...') #1 /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/AbstractDriver.php(64): Intervention\Image\AbstractDecoder->init('/var/www/trial/...') #2 /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/ImageManager.php(50): Intervention\Image\AbstractDriver->init('/var/www/trial/...') #3 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(206): Intervention\Image\ImageManager->make('/var/www/trial/...') #4 /var/www/trial/dropzone/app/routes.php(34): Illuminate\Support\Facades\Facade::__callStatic('make', Array) #5 /var/www/trial/dropzone/app/routes.php(34): Intervention\Image\Facades\Image::make('/var/www/trial/...') #6 [internal function]: {closure}() #7 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/trial/dropzone/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2015-01-16 06:20:35] log.ERROR: exception 'Intervention\Image\Exception\NotReadableException' with message 'Unable to read image from file (/var/www/trial/dropzone/public/uploads/evaluation-result-blok_2_-_research_methodology_1.xlsx).' in /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/Gd/Decoder.php:21 Stack trace: #0 /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/AbstractDecoder.php(248): Intervention\Image\Gd\Decoder->initFromPath('/var/www/trial/...') #1 /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/AbstractDriver.php(64): Intervention\Image\AbstractDecoder->init('/var/www/trial/...') #2 /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/ImageManager.php(50): Intervention\Image\AbstractDriver->init('/var/www/trial/...') #3 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(206): Intervention\Image\ImageManager->make('/var/www/trial/...') #4 /var/www/trial/dropzone/app/routes.php(34): Illuminate\Support\Facades\Facade::__callStatic('make', Array) #5 /var/www/trial/dropzone/app/routes.php(34): Intervention\Image\Facades\Image::make('/var/www/trial/...') #6 [internal function]: {closure}() #7 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/trial/dropzone/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2015-01-16 06:20:35] log.ERROR: exception 'Intervention\Image\Exception\NotReadableException' with message 'Unable to read image from file (/var/www/trial/dropzone/public/uploads/evaluation-result-learning_how_people_learn_1_.xlsx).' in /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/Gd/Decoder.php:21 Stack trace: #0 /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/AbstractDecoder.php(248): Intervention\Image\Gd\Decoder->initFromPath('/var/www/trial/...') #1 /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/AbstractDriver.php(64): Intervention\Image\AbstractDecoder->init('/var/www/trial/...') #2 /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/ImageManager.php(50): Intervention\Image\AbstractDriver->init('/var/www/trial/...') #3 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(206): Intervention\Image\ImageManager->make('/var/www/trial/...') #4 /var/www/trial/dropzone/app/routes.php(34): Illuminate\Support\Facades\Facade::__callStatic('make', Array) #5 /var/www/trial/dropzone/app/routes.php(34): Intervention\Image\Facades\Image::make('/var/www/trial/...') #6 [internal function]: {closure}() #7 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/trial/dropzone/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2015-01-16 06:20:36] log.ERROR: exception 'Intervention\Image\Exception\NotReadableException' with message 'Unable to read image from file (/var/www/trial/dropzone/public/uploads/evaluation-result-research_methodology_1.xlsx).' in /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/Gd/Decoder.php:21 Stack trace: #0 /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/AbstractDecoder.php(248): Intervention\Image\Gd\Decoder->initFromPath('/var/www/trial/...') #1 /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/AbstractDriver.php(64): Intervention\Image\AbstractDecoder->init('/var/www/trial/...') #2 /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/ImageManager.php(50): Intervention\Image\AbstractDriver->init('/var/www/trial/...') #3 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(206): Intervention\Image\ImageManager->make('/var/www/trial/...') #4 /var/www/trial/dropzone/app/routes.php(34): Illuminate\Support\Facades\Facade::__callStatic('make', Array) #5 /var/www/trial/dropzone/app/routes.php(34): Intervention\Image\Facades\Image::make('/var/www/trial/...') #6 [internal function]: {closure}() #7 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/trial/dropzone/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2015-01-16 06:20:36] log.ERROR: exception 'Intervention\Image\Exception\NotReadableException' with message 'Unable to read image from file (/var/www/trial/dropzone/public/uploads/evaluation-result-learning_how_people_learn_1_ (1).xlsx).' in /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/Gd/Decoder.php:21 Stack trace: #0 /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/AbstractDecoder.php(248): Intervention\Image\Gd\Decoder->initFromPath('/var/www/trial/...') #1 /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/AbstractDriver.php(64): Intervention\Image\AbstractDecoder->init('/var/www/trial/...') #2 /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/ImageManager.php(50): Intervention\Image\AbstractDriver->init('/var/www/trial/...') #3 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(206): Intervention\Image\ImageManager->make('/var/www/trial/...') #4 /var/www/trial/dropzone/app/routes.php(34): Illuminate\Support\Facades\Facade::__callStatic('make', Array) #5 /var/www/trial/dropzone/app/routes.php(34): Intervention\Image\Facades\Image::make('/var/www/trial/...') #6 [internal function]: {closure}() #7 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/trial/dropzone/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2015-01-16 06:22:49] log.ERROR: exception 'Intervention\Image\Exception\NotReadableException' with message 'Unable to read image from file (/var/www/trial/dropzone/public/uploads/TableTools.pdf).' in /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/Gd/Decoder.php:21 Stack trace: #0 /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/AbstractDecoder.php(248): Intervention\Image\Gd\Decoder->initFromPath('/var/www/trial/...') #1 /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/AbstractDriver.php(64): Intervention\Image\AbstractDecoder->init('/var/www/trial/...') #2 /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/ImageManager.php(50): Intervention\Image\AbstractDriver->init('/var/www/trial/...') #3 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(206): Intervention\Image\ImageManager->make('/var/www/trial/...') #4 /var/www/trial/dropzone/app/routes.php(34): Illuminate\Support\Facades\Facade::__callStatic('make', Array) #5 /var/www/trial/dropzone/app/routes.php(34): Intervention\Image\Facades\Image::make('/var/www/trial/...') #6 [internal function]: {closure}() #7 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/trial/dropzone/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2015-01-16 06:24:56] log.ERROR: exception 'Intervention\Image\Exception\NotReadableException' with message 'Unable to read image from file (/var/www/trial/dropzone/public/uploads/evaluation-result-learning_how_people_learn_1_ (1).xlsx).' in /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/Gd/Decoder.php:21 Stack trace: #0 /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/AbstractDecoder.php(248): Intervention\Image\Gd\Decoder->initFromPath('/var/www/trial/...') #1 /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/AbstractDriver.php(64): Intervention\Image\AbstractDecoder->init('/var/www/trial/...') #2 /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/ImageManager.php(50): Intervention\Image\AbstractDriver->init('/var/www/trial/...') #3 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(206): Intervention\Image\ImageManager->make('/var/www/trial/...') #4 /var/www/trial/dropzone/app/routes.php(34): Illuminate\Support\Facades\Facade::__callStatic('make', Array) #5 /var/www/trial/dropzone/app/routes.php(34): Intervention\Image\Facades\Image::make('/var/www/trial/...') #6 [internal function]: {closure}() #7 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/trial/dropzone/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2015-01-16 06:24:56] log.ERROR: exception 'Intervention\Image\Exception\NotReadableException' with message 'Unable to read image from file (/var/www/trial/dropzone/public/uploads/evaluation-result-learning_how_people_learn_1_.xlsx).' in /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/Gd/Decoder.php:21 Stack trace: #0 /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/AbstractDecoder.php(248): Intervention\Image\Gd\Decoder->initFromPath('/var/www/trial/...') #1 /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/AbstractDriver.php(64): Intervention\Image\AbstractDecoder->init('/var/www/trial/...') #2 /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/ImageManager.php(50): Intervention\Image\AbstractDriver->init('/var/www/trial/...') #3 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(206): Intervention\Image\ImageManager->make('/var/www/trial/...') #4 /var/www/trial/dropzone/app/routes.php(34): Illuminate\Support\Facades\Facade::__callStatic('make', Array) #5 /var/www/trial/dropzone/app/routes.php(34): Intervention\Image\Facades\Image::make('/var/www/trial/...') #6 [internal function]: {closure}() #7 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/trial/dropzone/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2015-01-16 06:31:07] log.ERROR: exception 'ErrorException' with message 'Undefined variable: core' in /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/Gd/Decoder.php:50 Stack trace: #0 /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/Gd/Decoder.php(50): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/trial/...', 50, Array) #1 /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/AbstractDecoder.php(248): Intervention\Image\Gd\Decoder->initFromPath('/var/www/trial/...') #2 /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/AbstractDriver.php(64): Intervention\Image\AbstractDecoder->init('/var/www/trial/...') #3 /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/ImageManager.php(50): Intervention\Image\AbstractDriver->init('/var/www/trial/...') #4 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(206): Intervention\Image\ImageManager->make('/var/www/trial/...') #5 /var/www/trial/dropzone/app/routes.php(34): Illuminate\Support\Facades\Facade::__callStatic('make', Array) #6 /var/www/trial/dropzone/app/routes.php(34): Intervention\Image\Facades\Image::make('/var/www/trial/...') #7 [internal function]: {closure}() #8 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #9 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #10 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #11 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #13 /var/www/trial/dropzone/public/index.php(49): Illuminate\Foundation\Application->run() #14 {main} [] []
[2015-01-16 06:31:57] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to a member function setFileInfoFromPath() on a non-object' in /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/Gd/Decoder.php:52 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2015-01-16 06:31:58] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to a member function setFileInfoFromPath() on a non-object' in /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/Gd/Decoder.php:52 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2015-01-16 06:32:18] log.ERROR: exception 'ErrorException' with message 'Undefined variable: image' in /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/Gd/Decoder.php:54 Stack trace: #0 /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/Gd/Decoder.php(54): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/trial/...', 54, Array) #1 /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/AbstractDecoder.php(248): Intervention\Image\Gd\Decoder->initFromPath('/var/www/trial/...') #2 /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/AbstractDriver.php(64): Intervention\Image\AbstractDecoder->init('/var/www/trial/...') #3 /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/ImageManager.php(50): Intervention\Image\AbstractDriver->init('/var/www/trial/...') #4 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(206): Intervention\Image\ImageManager->make('/var/www/trial/...') #5 /var/www/trial/dropzone/app/routes.php(34): Illuminate\Support\Facades\Facade::__callStatic('make', Array) #6 /var/www/trial/dropzone/app/routes.php(34): Intervention\Image\Facades\Image::make('/var/www/trial/...') #7 [internal function]: {closure}() #8 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #9 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #10 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #11 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #13 /var/www/trial/dropzone/public/index.php(49): Illuminate\Foundation\Application->run() #14 {main} [] []
[2015-01-16 06:32:18] log.ERROR: exception 'ErrorException' with message 'Undefined variable: image' in /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/Gd/Decoder.php:54 Stack trace: #0 /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/Gd/Decoder.php(54): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/trial/...', 54, Array) #1 /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/AbstractDecoder.php(248): Intervention\Image\Gd\Decoder->initFromPath('/var/www/trial/...') #2 /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/AbstractDriver.php(64): Intervention\Image\AbstractDecoder->init('/var/www/trial/...') #3 /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/ImageManager.php(50): Intervention\Image\AbstractDriver->init('/var/www/trial/...') #4 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(206): Intervention\Image\ImageManager->make('/var/www/trial/...') #5 /var/www/trial/dropzone/app/routes.php(34): Illuminate\Support\Facades\Facade::__callStatic('make', Array) #6 /var/www/trial/dropzone/app/routes.php(34): Intervention\Image\Facades\Image::make('/var/www/trial/...') #7 [internal function]: {closure}() #8 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #9 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #10 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #11 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #13 /var/www/trial/dropzone/public/index.php(49): Illuminate\Foundation\Application->run() #14 {main} [] []
[2015-01-16 06:32:53] log.ERROR: exception 'Intervention\Image\Exception\NotReadableException' with message 'Unable to read image type. GD driver is only able to decode JPG, PNG or GIF files.' in /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/Gd/Decoder.php:44 Stack trace: #0 /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/AbstractDecoder.php(248): Intervention\Image\Gd\Decoder->initFromPath('/var/www/trial/...') #1 /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/AbstractDriver.php(64): Intervention\Image\AbstractDecoder->init('/var/www/trial/...') #2 /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/ImageManager.php(50): Intervention\Image\AbstractDriver->init('/var/www/trial/...') #3 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(206): Intervention\Image\ImageManager->make('/var/www/trial/...') #4 /var/www/trial/dropzone/app/routes.php(34): Illuminate\Support\Facades\Facade::__callStatic('make', Array) #5 /var/www/trial/dropzone/app/routes.php(34): Intervention\Image\Facades\Image::make('/var/www/trial/...') #6 [internal function]: {closure}() #7 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/trial/dropzone/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2015-01-16 06:50:43] log.ERROR: exception 'Intervention\Image\Exception\NotReadableException' with message 'Unable to read image from file (/var/www/trial/dropzone/public/uploads/evaluation-result-research_methodology_1.xlsx).' in /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/Gd/Decoder.php:21 Stack trace: #0 /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/AbstractDecoder.php(248): Intervention\Image\Gd\Decoder->initFromPath('/var/www/trial/...') #1 /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/AbstractDriver.php(64): Intervention\Image\AbstractDecoder->init('/var/www/trial/...') #2 /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/ImageManager.php(50): Intervention\Image\AbstractDriver->init('/var/www/trial/...') #3 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(206): Intervention\Image\ImageManager->make('/var/www/trial/...') #4 /var/www/trial/dropzone/app/routes.php(34): Illuminate\Support\Facades\Facade::__callStatic('make', Array) #5 /var/www/trial/dropzone/app/routes.php(34): Intervention\Image\Facades\Image::make('/var/www/trial/...') #6 [internal function]: {closure}() #7 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/trial/dropzone/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2015-01-16 06:54:07] log.ERROR: exception 'Intervention\Image\Exception\NotReadableException' with message 'Unable to read image from file (/var/www/trial/dropzone/public/uploads/evaluation-result-research_methodology_1.xlsx).' in /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/Gd/Decoder.php:21 Stack trace: #0 /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/AbstractDecoder.php(248): Intervention\Image\Gd\Decoder->initFromPath('/var/www/trial/...') #1 /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/AbstractDriver.php(64): Intervention\Image\AbstractDecoder->init('/var/www/trial/...') #2 /var/www/trial/dropzone/vendor/intervention/image/src/Intervention/Image/ImageManager.php(50): Intervention\Image\AbstractDriver->init('/var/www/trial/...') #3 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(206): Intervention\Image\ImageManager->make('/var/www/trial/...') #4 /var/www/trial/dropzone/app/routes.php(34): Illuminate\Support\Facades\Facade::__callStatic('make', Array) #5 /var/www/trial/dropzone/app/routes.php(34): Intervention\Image\Facades\Image::make('/var/www/trial/...') #6 [internal function]: {closure}() #7 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/trial/dropzone/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/trial/dropzone/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
